python/cugraph/cugraph/community/ktruss_subgraph.py

```diff
@@ -65,8 +65,6 @@ def ktruss_subgraph(
     """
     Returns the K-Truss subgraph of a graph for a specific k.
 
-    NOTE: this function is currently not available on CUDA 11.4 systems.
```

Comment:
This is a "special" note, as it triggers the `skip_docstring` function.

This is the only remaining CUDA incompatibility that we mark in this way. Can we remove `skip_docstring` entirely?

https://github.com/rapidsai/cugraph/blob/2c957142ebfbe6c2220ec13e7d48142312001f93/python/cugraph/cugraph/tests/docs/test_doctests.py#L122

Reply:
There are actually 4 PRs against the one CUDA 11 issues.  I just put the close in this PR that was planned to be the last

Reply:
Merged @bdice's change from #5177 which updates skip_docstring to remove the version logic.  We can always add it back later if we have a case for that... but hopefully we're learning not to do that :-)